tasks/task1/task1_1.js

```diff
@@ -0,0 +1,8 @@
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('readable', function() {
+    let chunk;
+    while ((chunk = process.stdin.read()) !== null) {
```

Comment:
Consider creating separate functions for process.stdin.read and process.stdin.write, e.x readFunc and writeFunc, to abstract away these operations.